resources/customVisuals/VisualDetail.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="zxx">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta content="width=device-width,initial-scale=1.0,maximum-scale=1.0" name="viewport">
+
+    <!-- Open Graph -->
+    <meta property="og:title" content="Power BI Custom Visual | MAQ Software" />
+    <meta property="og:type" content="website" />
+    <!-- <meta property="og:image" content="/images/expertise/ai-main" /> -->
+    <!-- <meta property="og:description"
```

Comment:
Let's add description here like 
Expand your reporting capabilities with our certified Power BI custom visuals. If you need help choosing the right visual for your report, try our visual guide.

Reply:
Added description